src/main/java/calculator/calculator/StringExpression.java

```diff
@@ -0,0 +1,25 @@
+package calculator.calculator;
+
+import java.util.regex.Pattern;
+
+public class StringExpression {
+
+    private final String str;
+    private static final Pattern patten = Pattern.compile("^[0-9+*-/\\s]*$");
```

Comment:
상수는 대문자로 선언 해주시면 됩니다 😄 

```suggestion
    private static final Pattern PATTERN = Pattern.compile("^[0-9+*-/\\s]*$");
```